packages/cli/src/commands/init/init.ts

```diff
@@ -56,6 +58,30 @@ interface TemplateOptions {
   installCocoaPods?: string | boolean;
 }
 
+const YARN_VERSION = '3.6.4';
+
+const bumpYarnVersion = async (silent: boolean, root: string) => {
+  try {
+    let yarnVersion = semver.parse(getYarnVersionIfAvailable());
+    if (yarnVersion && semver.major(yarnVersion) === 1) {
+      await executeCommand('yarn', ['set', 'version', YARN_VERSION], {
+        root,
+        silent,
+      });
```

Comment:
Just to check my understanding here - this will install the requested version via corepack if it isn't currently on the machine? What if corepack isn't enabled?